parm/config/gfs/config.anal

```diff
@@ -150,4 +150,10 @@ if [[ ${RUN_ENVIR} == "emc" ]]; then
   #   fi
 fi
 
+# Flag to turn on (.true.) or off (.false.) the infrared cloud and aerosol detection software
+# for AIRS, CrIS, and IASI.  Default is .false.
+export AIRS_CADS=${AIRS_CADS:-".false."}
```

Comment:
g-w & NCO question

Should we have
```
export AIRS_CADS=".false."
export CRIS_CADS=".false."
export IASI_CADS=".false."
```
or is it acceptable set a default value here and allow this default to be overridden by setting `*_CADS` variables elsewhere?

What do implementation standards or EE2 require?

Reply:
This should be *the* location, so just have the values without accepting pre-defined values.

Reply:
Variables set to `.false.` at [bbc0bcf](https://github.com/NOAA-EMC/global-workflow/pull/2540/commits/bbc0bcf707b2da951fc25c5a38b7c594e2632f73)